### PR TITLE
build: add .renkulfsignore and docs

### PR DESCRIPTION
**Pull request by emmjab**
closes: #74 

changes required by: SwissDataScienceCenter/renku-python#1210

### CODE REVIEW

File: R-minimal/.renkulfsignore [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# WARNING: editing this file will impact the behavior of renku commands
+# by telling renku to *not track* listed files with git LFS.
+#
+# By default, renku commands track all files above the renku configured lfs_threshold
+# with git LFS to prevent accidentally checking in large files (default=100kb).
+# You can update this threshold by calling renku config lfs_threshold <size>kb
```
Comment (rokroskar) [inline R-minimal/.renkulfsignore:6]: put the command in quotes or on another line?

File: R-minimal/.renkulfsignore [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+#
+# Use this file to specify particular paths regardless of the threshold size.
```
Comment (rokroskar) [inline R-minimal/.renkulfsignore:8]:
```suggestion
# Use this file to specify particular paths regardless of their size.
```

File: R-minimal/.renkulfsignore (hunk 1 above)
Comment (Panaetius) [inline R-minimal/.renkulfsignore:6]:
```suggestion
# You can update this threshold by calling 'renku config lfs_threshold <size>kb'
```

On a side note, units that work for this flag are: b, k, M, G, T, P, E, Z, Y, ki, Mi, Gi, Gi, Ti, Pi, Ei, Zi, Yi (going a bit overboard I guess but found a list of SI and other units online and used that). I'll add `b` variations as well (kb, mb, gb, tb)

Comment (emmjab) [reply R-minimal/.renkulfsignore]: oh good to know! totally just assumed kb. but since you're adding it i won't change this?

**Pull request merged**